_includes/doc.tsx

```diff
@@ -19,6 +19,14 @@ export default function Page(props: Lume.Data, helpers: Lume.Helpers) {
     throw new Error("Missing sidebar for " + props.url);
   }
 
+  function displayDate(date: string) {
+    return new Date(date).toLocaleDateString("en-US", {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+    });
+  }
```

Comment:
I'd prefer to render this as YYYY-mm-dd

also I think this should take a `Date` arg rather than string

```ts
function displayDate(date: Date): string {
  return date.toISOString().slice(0, 10);
}
```